<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramsPage</class>
 <widget class="QWidget" name="ProgramsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="algo5Push">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-e.png</normaloff>:/images/res/program-e.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="algo7Push">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-g.png</normaloff>:/images/res/program-g.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="algo6Push">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-f.png</normaloff>:/images/res/program-f.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="algo11Push">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-k.png</normaloff>:/images/res/program-k.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="algo10Push">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-j.png</normaloff>:/images/res/program-j.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="op1Box">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Operator 1</string>
   </property>
   <widget class="OperatorWidget" name="op1Widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>16</y>
      <width>420</width>
      <height>260</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="algo1Push">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-a.png</normaloff>:/images/res/program-a.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="algo8Push">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-h.png</normaloff>:/images/res/program-h.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="algo2Push">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-b.png</normaloff>:/images/res/program-b.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="algo3Push">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-c.png</normaloff>:/images/res/program-c.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="algo12Push">
   <property name="geometry">
    <rect>
     <x>825</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-l.png</normaloff>:/images/res/program-l.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="algo13Push">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-m.png</normaloff>:/images/res/program-m.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="op3Box">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Operator 3</string>
   </property>
   <widget class="OperatorWidget" name="op3Widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>16</y>
      <width>420</width>
      <height>260</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="programBox">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>0</y>
     <width>141</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>Program</string>
   </property>
   <widget class="QPushButton" name="i1Push">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="i2Push">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="i3Push">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="i4Push">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="i6Push">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="i7Push">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="i8Push">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="i5Push">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="ProgramWidget" name="programWidget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>150</y>
      <width>141</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="Led" name="i1Led" native="true">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="Led" name="i2Led" native="true">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="Led" name="i3Led" native="true">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="Led" name="i4Led" native="true">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="Led" name="i5Led" native="true">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>90</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="Led" name="i6Led" native="true">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>90</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="Led" name="i7Led" native="true">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>90</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="Led" name="i8Led" native="true">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>90</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="algo9Push">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-i.png</normaloff>:/images/res/program-i.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="op2Box">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Operator 2</string>
   </property>
   <widget class="OperatorWidget" name="op2Widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>16</y>
      <width>420</width>
      <height>260</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="algo4Push">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>590</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/program-d.png</normaloff>:/images/res/program-d.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="op4Box">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Operator 4</string>
   </property>
   <widget class="OperatorWidget" name="op4Widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>16</y>
      <width>420</width>
      <height>260</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OperatorWidget</class>
   <extends>QWidget</extends>
   <header>operatorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProgramWidget</class>
   <extends>QWidget</extends>
   <header>programwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Led</class>
   <extends>QWidget</extends>
   <header>led.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
